tests/unit/test_models.py

```diff
@@ -0,0 +1,41 @@
+import github3
+import pytest
+import mock
+from .helper import create_example_data_helper
+
+get_user_key_example_data = create_example_data_helper('user_key_example')
+
+
+class TestGitHubObject:
+    """Test methods on GitHubObject class."""
+
+    def test_getattr(self):
+        """Test access to JSON data if attribute is not explicitly set."""
+
+        key = github3.users.Key(get_user_key_example_data())
+        # verified attribute is not set in _update_attributes for Key class
+        with pytest.raises(KeyError):
+            key.__dict__['verified']
+
+        assert key.verified is True
+
+    def test_getattr_attribute_not_in_json(self):
+        """Test AttributeError is raised when attribute is not in JSON."""
+        key = github3.users.Key(get_user_key_example_data())
+        with pytest.raises(AttributeError):
+            key.fakeattribute
```

Comment:
In the updated tests, maybe call this "missingattribute" to be very explicit about the fact you don't want it in the example data. :)